web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -0,0 +1,64 @@
+.author-body {
+  .cover-image {
+    height: 250px;
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    @include breakpoint($lap) {
+      height: 350px;
+    }
+  }
+
+  .author-container {
+    margin-bottom: $unit *4;
+    padding: $unit $unit*3;
+    padding-left: 0;
+    background-color: $light-gray;
+
+    @include breakpoint($lap) {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .author-info-container {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+
+      @include breakpoint($lap) {
+        flex-direction: row;
+      }
+
+      .profile-image {
+        width: 50%;
```

Comment:
Make the image bigger on mobile (75%) looks good.

Reply:
done